hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -444,6 +450,13 @@ private enum State {
 
   private final boolean isSecurityEnabled;
 
+
+  /** A list of property that are reconfigurable at runtime. */
+  private final TreeSet<String> reconfigurableProperties = Sets.newTreeSet(
+      Lists.newArrayList(
+          OZONE_ADMINISTRATORS
+  ));
```

Comment:
Please use `ImmutableSortedSet` instead.  Also, please declare as `SortedSet` instead of the implementation.